### PR TITLE
Kokkos/arm ppc

### PR DESCRIPTION
**Pull request by vincentmr**
Add LKokkos - MacOS ARM & Linux ARM/PPC wheels.

### CODE REVIEW

**Review by mlxd — COMMENTED** [review]
Can we temporarily enable these builds in the PR to see the end to end wheel compilations? Same with caching the output wheels.

**Comment by vincentmr** [comment]
> Can we temporarily enable these builds in the PR to see the end to end wheel compilations? Same with caching the output wheels.

Yes, you may have a look at the builds of commit [8ce356f](https://github.com/PennyLaneAI/Lightning-Unification/pull/24/commits/8ce356f97506b165d69bbb519bf252b1d40173af). The next two commits restore the usual triggers.

**Comment by mlxd** [comment]
> > Can we temporarily enable these builds in the PR to see the end to end wheel compilations? Same with caching the output wheels.
> 
> Yes, you may have a look at the builds of commit [8ce356f](https://github.com/PennyLaneAI/Lightning-Unification/pull/24/commits/8ce356f97506b165d69bbb519bf252b1d40173af). The next two commits restore the usual triggers.

Thanks. I cannot seem to see the wheels though. Do you have a link to where the artifacts are saved?

**Comment by vincentmr** [comment]
> I cannot seem to see the wheels though.

You're right, I hadn't changed the upload-artifact trigger. Let me do that then.

File: .github/workflows/wheel_linux_aarch64.yml [hunk 1]
```diff
@@ -150,7 +161,7 @@ jobs:
           regex: '.*[0-9]+.[0-9]+.[0-9]+[-_]?rc[0-9]+'
 
       - uses: actions/upload-artifact@v2
-        if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' }}
```
Comment (AmintorDusko) [inline .github/workflows/wheel_linux_aarch64.yml:153]: Should we change this back to `release`?

File: .github/workflows/wheel_linux_ppc64le.yml [hunk 2]
```diff
@@ -6,6 +6,7 @@ name: Wheel::Linux::PowerPC
 # **Who does it impact**: Wheels to be uploaded to PyPI.
 
 on:
+  pull_request:
```
Comment (AmintorDusko) [inline .github/workflows/wheel_linux_ppc64le.yml:9]: does this need to be removed?

File: .github/workflows/wheel_linux_ppc64le.yml [hunk 3]
```diff
@@ -151,7 +155,7 @@ jobs:
           regex: '.*[0-9]+.[0-9]+.[0-9]+[-_]?rc[0-9]+'
 
       - uses: actions/upload-artifact@v2
-        if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' || steps.rc_build.outputs.match != ''}}
```
Comment (AmintorDusko) [inline .github/workflows/wheel_linux_ppc64le.yml:154]: should be `release` too, right?

**Review by AmintorDusko — COMMENTED** [review]
I left some comments regarding some changes that you probably did to test the wheels.

**Review by AmintorDusko — APPROVED** [review]
Looks good to me.
Before you merge, could you please re-enable the upload of the artifacts and double(triple?)-check if the wheels still make sense?
[PR 22](https://github.com/PennyLaneAI/Lightning-Unification/pull/22) changed how the wheels should look. Let me know if you need any support on that.

**Pull request merged**